src/common-tests/UrlParameter.test.js

```diff
@@ -0,0 +1,18 @@
+import { describe, expect } from "@jest/globals";
+import { getRequestedUrl } from "../components/OSCALLoader";
+import { urlParameterTestUrl } from "../test-data/Urls";
+
+describe("Grabbing a URL Parameter from the Browser", () => {
+  test("No Url Parameter Exists", () => {
+    // The Url parameter should always initially be null.
+    expect(getRequestedUrl()).toBeNull();
+  });
+
+  test("Url Parameter Exists", () => {
+    const url = new URL(window.location);
+    url.searchParams.set("url", urlParameterTestUrl);
+    window.history.pushState({}, "", url);
+    expect(getRequestedUrl()).not.toBeNull();
+    expect(getRequestedUrl()).toEqual(urlParameterTestUrl);
```

Comment:
Is it needed to check that this is not `null` and that it equals `urlParameterTestUrl`?

Reply:
Yeah, I realize that now how the null check is unnecessary.